korman/exporter/decal.py

```diff
@@ -0,0 +1,174 @@
+            self._decal_managers[decal_name].append(decal_mgr.key)
+
+            # Certain decals are required to be squares
+            if decal_type in {"footprint_dry", "footprint_wet", "wake"}:
+                length, width = decal.length / 100.0, decal.width / 100.0
+            else:
+                length = max(decal.length, decal.width) / 100.0
+                width = max(decal.length, decal.width) / 100.0
+
+            image = decal.image
+            if image is None:
+                raise ExportError("'{}': decal manager '{}' has no image set", bo.name, decal_name)
+
+            blend = getattr(hsGMatState, decal.blend)
+            mats = exporter.mesh.material.export_print_materials(bo, image, name, blend)
+            decal_mgr.matPreShade, decal_mgr.matRTShade = mats
+
+            # Hardwired values from PlasmaMAX
+            decal_mgr.maxNumVerts = 1000
+            decal_mgr.maxNumIdx = 1000
+            decal_mgr.intensity = decal.intensity / 100.0
+            decal_mgr.gridSizeU = 2.5
+            decal_mgr.gridSizeV = 2.5
+            decal_mgr.scale = hsVector3(length, width, 1.0)
+
+            # Hardwired calculations from PlasmaMAX
+            if decal_type in {"footprint_dry", "footprint_wet", "bullet"}:
+                decal_mgr.rampEnd = 0.1
+                decal_mgr.decayStart = decal.life_span - (decal.life_span * 0.25)
+                decal_mgr.lifeSpan = decal.life_span
+            elif decal_type in {"puddle", "ripple", "torpedo", "wake"}:
+                decal_mgr.rampEnd = 0.25
+                life_span = decal.life_span if decal_type == "torpedo" else length / 2.0
+                decal_mgr.decayStart = life_span * 0.8
+                decal_mgr.lifeSpan = life_span
+            else:
+                raise RuntimeError()
+
+            # While any decal manager can be wet/dry, it really makes the most sense to only
+            # expose wet footprints. In the future, we could expose the plDynaDecalEnableMsg
+            # to nodes for advanced hacking.
+            decal_mgr.waitOnEnable = decal_type == "footprint_wet"
+            if decal_type in {"puddle", "ripple"}:
+                decal_mgr.wetLength = decal.wet_time
+                self._notifies[decal_name].extend((i.name for i in decal.wet_managers
+                                                          if i.enabled))
+
+            # UV Animations are hardcoded in PlasmaMAX. Any reason why we should expose this?
+            # I can't think of any presently... Note testing the final instance instead of the
+            # artist setting in case that gets overridden (puddle -> ripple)
+            if isinstance(decal_mgr, plDynaPuddleMgr):
+                decal_mgr.initUVW = hsVector3(0.5, 0.5, 0.5)
+                decal_mgr.finalUVW = hsVector3(1.0, 1.0, 1.0)
+            elif isinstance(decal_mgr, plDynaRippleMgr):
+                # wakes, torpedos, and ripples...
+                decal_mgr.initUVW = hsVector3(0.3, 0.3, 0.3)
```

Comment:
Likewise, these are [defaulted to 3.0](https://github.com/H-uru/Plasma/blob/master/Sources/Tools/MaxComponent/plFootPrintComponent.cpp#L704), not 0.3